GooglePlayInstant/Editor/PlayInstantQuickDeployBuild.cs

```diff
@@ -46,31 +45,38 @@ public static void BuildQuickDeployInstantGameApk()
                 locationPathName = QuickDeployConfig.Config.apkFileName,
                 target = BuildTarget.Android,
                 options = BuildOptions.None
+                // TODO: include asset bundle manifest path in options.
             };
 
 
-            if (!ProjectIsUsingIl2cpp() || !PlayerSettings.stripEngineCode)
+            if (!ProjectIsUsingIl2cpp())
             {
-                var enableIl2cppAndEngineStripping = EditorUtility.DisplayDialog(
-                    "IL2CPP or engine stripping not enabled",
-                    "Your project is not using IL2CPP scripting runtime, or engine stripping is not enabled. Would " +
-                    "you like to build the APK with IL2CPP and Engine Stripping to improve game " +
-                    "performance and reduce APK size?", "Yes", "No");
+                var enableIl2cppAndEngineStripping = EditorUtility.DisplayDialog("Scripting backend recommendation",
+                    "Current Scripting runtime is Mono. Want IL2CPP with engine stripping?", "Yes", "No");
```

Comment:
Please make "Yes" and "No" constants and reuse below.

Reply:
Resolved